css/elements/image.css

```diff
@@ -0,0 +1,9 @@
+.img-responsive{
+    width: 100%;
+    height: auto;
+}
+
+.img-rounded{
+    border-radius: 50%;
```

Comment:
in our component library we using this a lot of the times , you can make it a separate variable.

Reply:
Thanks will do that